utils/upload_everything/upload_movies.py

```diff
@@ -0,0 +1,304 @@
+results in the time range exceeds 100, only the first 100 events will have their movies requested. Setting the time range
+chunks to 30 minutes prevents exceeding 100 events.
+
+Execute with:
+python upload_movies.py --cameras_filename your_cameras_filename.csv --time_range_filename your_time_range_filename.csv --token your_access_token --device_ids "camio_box_device_id_1" "camio_box_device_id_2" --max_wait_seconds 600
+
+See help with:
+python upload_movies.py --help
+
+cameras_filename is a csv with a header row and three required columns like this,
+where the rows without is_requested value TRUE are skipped:
+
+user_email,camera,is_requested
+sanmateo@camiolog.com,Front Entrance,TRUE
+sanmateo@camiolog.com,Back Exit,TRUE
+
+
+time_range_filename is a csv with a header row and two required columns like this,
+where each time range should span less than 800 Events (for reference, the max number
+of Events/hour/camera is typically 180):
+
+start_time,end_time
+7pm PT January 31,7:30pm PT January 31st
+7:30pm PT January 31,8:00pm PT January 31st
+
+You can obtain your_access_token from:
+https://camio.com/settings/integrations/#api
+
+The stdout is a CSV with five columns like this example, where the search_url can be used to view the results the requested
+uploads are completed:
+
+python upload_movies.py --cameras_filename cameras.csv --time_range_filename time-ranges.csv --token YOURTOKEN --device_ids "camio_box_device_id_1" "camio_box_device_id_2" --max_wait_seconds 600 | tee output_upload_movies.csv
+
+timestamp,api_request_url,status,upload_commands_count,search_url,message
+2024-02-05T14:56:39.197221,https://camio.com/api/search?text=sanmateo%40camiolog.com+Front+East+7pm+PT+January+31st+to+8pm+PT+January+31st+all+tag%3Abox,200,2,0,https://camio.com/app/#search;q=sanmateo%40camiolog.com+Front+East+7pm+PT+January+31st+to+8pm+PT+January+31st+all,success requesting uploads
+2024-02-05T14:56:40.197221,https://camio.com/api/search?text=sanmateo%40camiolog.com+Front+East+8pm+PT+January+31st+to+9pm+PT+January+31st+all+tag%3Abox,200,2,1,https://camio.com/app/#search;q=sanmateo%40camiolog.com+Front+East+8pm+PT+January+31st+to+9pm+PT+January+31st+all,success requesting uploads
+2024-02-05T14:56:41.197221,https://camio.com/api/search?text=sanmateo%40camiolog.com+Front+West+7pm+PT+January+31st+to+8pm+PT+January+31st+all+tag%3Abox,200,2,2,https://camio.com/app/#search;q=sanmateo%40camiolog.com+Front+West+7pm+PT+January+31st+to+8pm+PT+January+31st+all,success requesting uploads
+2024-02-05T14:56:42.197221,https://camio.com/api/search?text=sanmateo%40camiolog.com+Front+West+8pm+PT+January+31st+to+9pm+PT+January+31st+all+tag%3Abox,200,2,3,https://camio.com/app/#search;q=sanmateo%40camiolog.com+Front+West+8pm+PT+January+31st+to+9pm+PT+January+31st+all,success requesting uploads
+
+Only the tag:box upload request query text is shown when using the --dry_run argument like this:
```

Comment:
just for doc clarity remove reference to tag:box since we don't use that for this movie retrieval.
Also make a note that the Events must have been uploaded to the cloud (e.g. with upload_everything.py) prior to running this script.